cluster-autoscaler/hack/update-vendor.sh

```diff
@@ -50,8 +50,13 @@ rm -rf ${WORKDIR}
 
 for MOD in "${MODS[@]}"; do
     V=$(
-        go mod download -json "${MOD}@kubernetes-${VERSION}" |
-        sed -n 's|.*"Version": "\(.*\)".*|\1|p'
+        GOMOD=$(go mod download -json "${MOD}@kubernetes-${VERSION}")
+        retval=$?
+        if [ $retval -ne 0 ]; then
+            echo "Error downloading module."
```

Comment:
Can we include the module name in the message?

Reply:
Hmm added the module in the message just how, doesn't looks better.